<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitorwindow</class>
 <widget class="QMainWindow" name="Monitorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>489</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>489</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>489</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monitorwindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
    background-image: url(:/new/Background/bg4.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="Status_sensor2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="Status_sensor1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Light Sensor :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor4_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QProgressBar" name="Status_sensor3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature Sensor (*C) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gas Sensor :</string>
    </property>
   </widget>
   <widget class="QLabel" name="IR1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#bulbglow
{
	border-image : url(:/basic/images/bulb_off.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IR2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#bulbglow
{
	border-image : url(:/basic/images/bulb_off.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IR3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#bulbglow
{
	border-image : url(:/basic/images/bulb_off.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor4_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Sensor4_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor4_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensor4_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
